proto/server/datastore/datastore.proto

```diff
@@ -189,14 +189,21 @@ message BySelectors {
     MatchBehavior match = 2;
 }
 
+message Pagination {
+    uint32 token = 1;
```

Comment:
i think a string "token" is more flexible... what do others think?

Reply:
Agree that was my first approach too

Reply:
My only gripe is that the sql plugin code is a little pointier when this is a string because we have to convert it and check for errors in a spot that would otherwise be error-free

Reply:
yes, I added a parser, to prevent it to happens, basically parse to int and put that int into select, and an error is returned in case parse is not valid